src/handlers/commands.js

```diff
@@ -1,21 +1,18 @@
 const { getPermissionLevel } = require("../constants/index.js"), { deleteMessages } = require("./counting.js"), fs = require("fs"), config = require("../../config.json");
 
 // loading commands
-const
-  commands = new Map(),
-  aliases = new Map(),
-  statics = require("../commands/_static.json"),
-  commandLoader = new Promise(resolve => fs.readdir("./src/commands/", (err, files) => {
-    if (err) return console.log(err);
-    for (const file of files) if (file.endsWith(".js")) {
-      const commandFile = require(`../commands/${file}`), fileName = file.replace(".js", "");
-      if (config.isPremium || !commandFile.premiumOnly) {
-        commands.set(fileName, commandFile);
-        if (commandFile.aliases) for (const alias of commandFile.aliases) aliases.set(alias, fileName);
-      }
+const commands = new Map(), aliases = new Map(), statics = require("../commands/_static.json");
+fs.readdir("./src/commands/", (err, files) => {
+  if (err) return console.log(err);
+  for (const file of files) if (file.endsWith(".js")) {
+    const commandFile = require(`../commands/${file}`), fileName = file.replace(".js", "");
+    if (config.isPremium || !commandFile.premiumOnly) {
+      commands.set(fileName, commandFile);
+      if (commandFile.aliases) for (const alias of commandFile.aliases) aliases.set(alias, fileName);
     }
-    resolve();
```

Comment:
Resolve is not defined here because it is not a promise

Reply:
Fixed in https://github.com/countr/countr/pull/83/commits/c6b4fb30f053d84a94eed6278523f4ccb4e86b43